### ISSUE TITLE
align Types does not work with CONSTANTS 😢 

### ISSUE THREAD

**Issue by Ennowulff** [fidley/ABAPQuickFix#45]
Unfortunately the quick "Align TYPEs" fix does not work with CONSTANTS.

**Comment by Ennowulff** [comment]
```
INTERFACE zif_test_2
  PUBLIC .

  CONSTANTS demo TYPE c LENGTH 1 VALUE 'D'.
  CONSTANTS verfifications TYPE c LENGTH 1 VALUE 'V'.
  CONSTANTS savemode TYPE c LENGTH 1 VALUE 'S'.

ENDINTERFACE.
```

![image](https://user-images.githubusercontent.com/75187288/202452858-c63ec381-535c-4d5f-937d-a800ff557459.png)

**Comment by Ennowulff** [comment]
PS: check spelling "Mehtods"! 😃

**Comment by fidley** [comment]
I think I know why. Put for example an DATA statement before constants. Then it should work.

I might have problem in the code for the first statements and last statements. I'll look into it in the evening

**Comment by Ennowulff** [comment]
_I think I know why. Put for example an DATA statement before constants. Then it should work._
confirmed.

don't bother. It's not really urgent... 😄

**Comment by fidley** [comment]
it will not let me sleep ;D

**Comment by fidley** [comment]
I've corrected this yesterday. I'll publish soon in on beta update site

**Comment by fidley** [comment]
You can update from beta site and check. I've added also help to the plugin.

**Comment by fidley** [comment]
solved with #47
